ci/scripts/cpp_checks.sh

```diff
@@ -42,6 +43,16 @@ if [[ -n "${SRF_MODIFIED_FILES}" ]]; then
       echo "  $f"
    done
 
+   # Check for `#pragma once` in any .hpp or .h file
+   if [[ "${SKIP_PRAGMA_CHECK}" == "" ]]; then
```

Comment:
Is there ever a reason where we would want to allow headers without `#pragma once`? Is this for something like old style ifdef guards?

Reply:
I was just following the pattern of the rest of the file that uses the skip variables to allow users to pick and choose which checks get run. I dont think you would ever want to skip it, but it doesnt hurt to have the option.